_rules/object-has-acessible-name-8fc3b6.md

```diff
@@ -38,11 +41,16 @@ The `object` element is not rendered for presentational purposes. If the `object
 
 ## Accessibility Support
 
-Non-supported media formats make screen readers render the text content of the element instead of other attributes.
+Non-supported media formats make screen readers render the text content of the element instead of other attributes. `object` elements without an accessible name are ignored by assistive technologies unless they have an [explicit role][].
 
 ## Background
 
-Testing that the [accessible name][] describes the purpose of the `object` element is not part of this rule and must be tested separately.
+This rule is testing that the following 'object' attributes are followed:
+
+- The purpose is in the [accessible name][] but is not part of this rule and must be tested separately; and 
```

Comment:
It's a bit weird to says that "this rule is testing the following stuff" and then list something that "is not part of the rule" 🤔

Reply:
Yeah - I could take that out of the list :) I was trying to find a grouping and failed miserably!